components/DocumentationCurrentPage.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+
+<article class="pt-6 lg:pt-0">
+    <h1 class="text-5xl font-bold text-gray-900 border-b border-slate-400 pb-4 mb-6 dark:text-slate-200 text-center">{{props.currentSubsection}}</h1>
+    <span v-if="props.renderedContent" v-html="props.renderedContent" class="mt-8 mx-auto leading-8 prose dark:prose-invert dark:prose-headings:text-MITRE-silver dark:prose-p:text-dark-text "></span>
+</article>
+<dl class="mt-12 mb-6 lg:mb-0 flex border-t border-slate-200 pt-6 dark:border-slate-800">
```

Comment:
I made this an issue as the DL tag is used throughout the site and seems to be common practice by other companies.  This will require some further research and be readdressed at a later point.